src/Collection.php

```diff
@@ -1517,4 +1517,36 @@ public function stats()
             'collstats' => $this->getName(),
         ));
     }
+
+    /**
+     * Rename collection x to y, with drop previously created target or not option
+     * Note: if dropTarget is false and collection existent, then command will fail
+     *
+     * @param string $target Target name of collection
+     * @param bool $dropTarget Whether to drop previously created collection
+     * @return bool
+     * @throws \Sokil\Mongo\Exception
+     * @throws \MongoException
+     */
+    public function renameCollection($target, $dropTarget = false)
+    {
+        // create admin db instance
+        $adminDb = new Database($this->getDatabase()->getClient(), 'admin');
```

Comment:
We getting an admin db context to only run `executeCommand`

Reply:
Creaning new instance is bed idea. 

This code do the same
```php
$this->getDatabase()->getClient()->getDatabase('admin');
```